dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -719,7 +719,7 @@ BEGIN
     GROUPBOX "Icones", IDC_STATIC, 5, 165, 230, 50, WS_TABSTOP
     LTEXT "Para todas as vistas, excepto miniaturas, pode alterar o ícone padrão 'pasta' para outro ícone.", IDC_STATIC, 15, 175, 210, 25
     ICON 0, IDC_FOLDERCUST_ICON, 175, 194, 32, 30
-    PUSHBUTTON "Alterar &icone.", IDC_FOLDERCUST_CHANGE_ICON, 15, 195, 75, 15
+    PUSHBUTTON "Alterar &icone", IDC_FOLDERCUST_CHANGE_ICON, 15, 195, 75, 15
```

Comment:
```suggestion
    PUSHBUTTON "Alterar &icone...", IDC_FOLDERCUST_CHANGE_ICON, 15, 195, 75, 15
```
with three dots actually, which signals another dialog will be opened.
See https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDC_FOLDERCUST_CHANGE_ICON